kartothek/api/utils.py

```diff
@@ -0,0 +1 @@
+from kartothek.core.utils import *  # noqa: F401, F403
```

Comment:
Do we really need `api.utils`? This is now a subset of `api.dataset`, isn't it?

Reply:
good point